lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -295,16 +299,21 @@ def common_path(paths)
           #         The consumer file accessor for which the headers need to be
           #         linked.
           #
+          # @param  [Symbol] visibility_scope
+          #         The visibility scope to produce header mappings for. If set to :public then the headers
+          #         are nested an additional level deep. For example, 'Pods/Headers/Public/PodA/PodA'.
+          #
           # @param  [Array<Pathname>] headers
           #         The absolute paths of the headers which need to be mapped.
           #
           # @return [Hash{Pathname => Array<Pathname>}] A hash containing the
           #         headers folders as the keys and the absolute paths of the
           #         header files as the values.
           #
-          def header_mappings(headers_sandbox, file_accessor, headers)
+          def header_mappings(headers_sandbox, file_accessor, headers, visibility_scope = :private)
```

Comment:
I am thinking on removing `visibilty_scope` from this method and instead pass in the initializer. Some header stores are explicitly created as private stores but yet this method talks about a `public` header scope.

Reply:
done.

Reply:
actually this comment is on the wrong file, I meant `header_store.rb` and yes I changed it there. This is fine to keep here.